syllabus/examples/training_scripts/cleanrl_procgen_plr.py

```diff
@@ -204,7 +205,10 @@ def fast_level_replay_evaluate(
 
     while -1 in eval_episode_rewards:
         with torch.no_grad():
-            eval_action, _, _, _ = policy.get_action_and_value(torch.Tensor(eval_obs).to(device), deterministic=False)
+            if evaluator:
+                eval_action, _, _, _ = evaluator.get_action_and_value_fn(eval_obs)
+            else:
+                eval_action, _, _, _ = policy.get_action_and_value(torch.Tensor(eval_obs).to(device), deterministic=False)
```

Comment:
I don't think you should need to modify this function at all, why do you use the evaluator here?